jquery_dom_selector_practice/js/exercise.js

```diff
@@ -85,7 +106,10 @@
  * Bonus: Try to do this with only one jQuery call. $("selector").css({attribute: "value", attribute: "value"});
  */
 
-
+$("#button3").click(question9);
+	function question9(){
+		$('p').css('font-family','Georgia').css('color', 'blue');
```

Comment:
Oh! (Sorry) and try and favor "cameCase" over "hyphen" style. In other words, use `'fontFamily'` instead of `'font-family'`